spec/models/user_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe User do
+  it { should have_many(:reviews) }
+  it { should have_many(:queued_videos).order("queued_videos.order") }
```

Comment:
Instead of `order("queued_videos.order")` you should be able to do `order(:order)` even tho i would call it differently like `position`.